raspberry-pi/4/default.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, pkgs, ...}:
+
+{
+  boot = {
+    kernelPackages = lib.mkDefault pkgs.linuxPackages_rpi4;
+    initrd.availableKernelModules = [ "usbhid" "usb_storage" "vc4" ];
+
+    loader = {
+      grub.enable = lib.mkDefault false;
+      generic-extlinux-compatible.enable = lib.mkDefault true;
+    };
+  };
+
+  nix.buildCores = 4;
```

Comment:
We no longer need this since by default it will use all cores already, see `man configuration.nix`:

```suggestion
```

Reply:
How about we set `nix.maxJobs = 4;`?

Reply:
Also not needed. The current default for maxJobs in nixos is `auto`, which means it will also use 4 cores on raspberrypi